### ISSUE TITLE
Gaia query does not return all sources for NGC 6819

### ISSUE THREAD

**Issue by jorgemarpa** [SSDataLab/psfmachine#20]
When using `psfmachine.TPFMachine` with the regular Gaia query (`get_gaia_sources`) there are missing sources in the result for NGC 6819(see Fig 1). 

```python
import lightkurve as lk
import psfmachine as pm

# TPF for NGC 6819 are local
fnames = glob.glob("../../data/kepler/stc/%s/q%02i/kplr*" % (cluster, quarter))
tpfs = lk.TargetPixelFileCollection([lk.read(f) for f in fnames])

ngc = pm.TPFMachine.from_TPFs(tpfs, do_sparse=True, dr=3)
```
![Fig 1](https://user-images.githubusercontent.com/10449555/122845273-2adeec00-d2b8-11eb-9e00-939a24c56599.png)

This does not happen when using `do_tiled_query` for the same sky area (see Fig 2, source table has not been cleaned, just for illustration purposes).

```python
from astropy.time import Time
from psfmachine.utils import do_tiled_query

epoch = Time(ngc.time[len(ngc.time) // 2], format="jd").jyear 
sources_2 = do_tiled_query(ngc.ra, ngc.dec, ngrid=(5, 5), magnitude_limit=18, epoch=epoch, dr=3)
```

![Fig 2](https://user-images.githubusercontent.com/10449555/122845302-421dd980-d2b8-11eb-84a6-dbd2e1c12a2c.png)

**Comment by jorgemarpa** [comment]
further investigations show that the centers for the cone search are off. The figure shows the cone search calculated by `_get_coord_and_query_gaia`

![image](https://user-images.githubusercontent.com/10449555/122852422-7055e600-d2c5-11eb-88d1-cb0901ef8675.png)

This also happens with "extended" TPFs, eg:

```python

tpfs2 = lk.search_targetpixelfile('KIC 3733346', mission='Kepler', 
                                 quarter=12, radius=2000, limit=400, 
                                 cadence='long').download_all(quality_bitmask=None)
```


![image](https://user-images.githubusercontent.com/10449555/122852514-954a5900-d2c5-11eb-9e34-d446c54f1ee2.png)
The shade circles are the cone searches. Note that for extended TPFs the circle does not cover the entire TPF.

**Comment by jorgemarpa** [comment]
Error found when converting pixels to RA Dec coordinates to calculate center and radius of cone search. Row and column are in the wrong order.

https://github.com/SSDataLab/psfmachine/blob/831be0ae04b05cf24f120a31440ee3d9bff634d0/src/psfmachine/tpf.py#L886
Fixed in PR #21

Example results:
Stack of various TPFs, including extended masks
![image](https://user-images.githubusercontent.com/10449555/122967978-e4cd6b00-d33f-11eb-8b69-4a1cd425f194.png)

Example of cluster field, cone search and results:
![image](https://user-images.githubusercontent.com/10449555/122968816-cae05800-d340-11eb-8bd6-fe5cec429e20.png)
![image](https://user-images.githubusercontent.com/10449555/122968646-9d93aa00-d340-11eb-8766-481986fff6e0.png)
